### PR TITLE
[GITHUB-1] Role to install Grafana server.

### PR DESCRIPTION
**Pull request by rsgoodall**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+language: python
+python: "2.7"
+
+branches:
+  only:
+    - develop
+    - master
+
+env:
+  - ANSIBLE_INSTALL_VERSION=2.2.3.0
```
Comment (lobsterdore) [inline .travis.yml]: I think we need the full list here:

```
  - ANSIBLE_INSTALL_VERSION=2.0.2.0
  - ANSIBLE_INSTALL_VERSION=2.1.5.0
  - ANSIBLE_INSTALL_VERSION=2.2.3.0
  - ANSIBLE_INSTALL_VERSION=2.3.1.0
```

File: defaults/main.yml [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+grafana:
+  environment: "{{ env if env is defined else 'dev' }}"
+  user: grafana
+  group: grafana
+  path:
+    log: /var/log/grafana/grafana.log
+    pid: /var/run/grafana-server.pid
+    data: /var/lib/grafana
+  version: 4.4.2
+
```
Comment (lobsterdore) [inline defaults/main.yml]: Can you delete the extra line here?

File: defaults/main.yml [hunk 3]
```diff
@@ -0,0 +1,12 @@
+---
+
+grafana:
+  environment: "{{ env if env is defined else 'dev' }}"
```
Comment (lobsterdore) [inline defaults/main.yml]: Is this var needed?

File: defaults/main.yml [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+  user: grafana
```
Comment (lobsterdore) [inline defaults/main.yml]: User should be between path and version (alpha ordering).

File: defaults/main.yml [hunk 5]
```diff
@@ -0,0 +1,12 @@
...
+  group: grafana
+  path:
+    log: /var/log/grafana/grafana.log
```
Comment (lobsterdore) [inline defaults/main.yml]: Can you alpha order these child keys as well.

File: tasks/configure.yml [hunk 6]
```diff
@@ -0,0 +1,17 @@
...
+    owner: "{{ grafana.user }}"
+    group: "{{ grafana.group }}"
+  register: grafana_config
+
+- name: Restart Grafana
+  become: yes
+  service:
+    name: grafana-server
+    state: restarted
+  when: grafana_config.changed and grafana.db.type == 'sqlite3'
```
Comment (lobsterdore) [inline tasks/configure.yml:18]: Can we get a small comment on why we have the when statement here?

Comment (rsgoodall) [reply tasks/configure.yml:18]: comment added.

**Review by lobsterdore — APPROVED** [review]
LGTM 👍

**Pull request merged**